automation/validate-pvc-name-stability.py

```diff
@@ -0,0 +1,92 @@
+#! python
+
+import logging
+import os
+import os.path
+import yaml
+import sys
+
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+
+
+def validatePVCNames(path, liveTemplates):
+    templates = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+    errors = []
+    for templateFilename in templates:
+        with open(os.path.join(path, templateFilename), 'r') as stream:
+            try:
+                template = yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
```

Comment:
We may want to add an `else` case here so causes of other errors are recorded also.